applications/SwimmingDEMApplication/tests/tests_python_scripts/candelier_scripts/candelier_analysis.py

```diff
@@ -100,4 +102,13 @@ def FinalizeSolutionStep(self):
                     coor_calculated[1] = node.X * sin + node.Y * cos
 
                 self.radial_error = self.results_database.CalculateError(self.time, coor_calculated)
-                self.error_time = self.time
\ No newline at end of file
+                self.error_time = self.time
+
+    def _CreateSolver(self):
+        import candelier_dem_solver as swimming_dem_solver
```

Comment:
full path?

Reply:
Ok, _CreateSolver was being overriden twice in the same file.
Apart from this silly bug, as @philbucher suggested I wrote
`import tests_python_scripts.candelier_scripts.candelier_dem_solver as sdem_solver`
instead of 
`import candelier_dem_solver as sdem_solver`
and it works. 
I don't really understand how it works, as the candelier_dem_solver.py file is in the same folder as the candelier_analysis.py file.